doc/rst/source/explain_-T_rose.rst_

```diff
@@ -1,5 +1,5 @@
 **-Td**\ [**g**\ \|\ **j**\ \|\ **J**\ \|\ **n**\ \|\ **x**]\ *refpoint*\ **+w**\ *width*\ [**+f**\ [*level*]]\ [**+j**\ *justify*]\ [**+l**\ *w,e,s,n*][**+o**\ *dx*\ [/*dy*]]
-    **-Td** draws a map directional rose on the map at the location defined by
+    Draws a map directional rose on the map at the location defined by
```

Comment:
```suggestion
    Draw a map directional rose on the map at the location defined by
```

Reply:
I dont have permission to commit those suggestions, apparently.

Reply:
Me too. We have to wait for @tparker37 to respond, or we can open a separated PR.